### ISSUE TITLE
Eliminate Background Scrolling on Fullscreen dialog Component

### ISSUE THREAD

**Issue by danieltisdale** [MerlinLabs/styled-material-components#276]
Fullscreen dialog component needs to eliminate background scrolling when open.

See video here:
![full screen dialog scrolling issue](https://user-images.githubusercontent.com/7127838/39934557-d4c92a92-550b-11e8-9e80-83a317c348de.gif)

**Comment by petegivens** [comment]
@danieltisdale I can't replicate this anymore. Are you still seeing this in SMC 0.0.52?

**Comment by danieltisdale** [comment]
@petegivens Yeah I don't see the issue in Chrome dev tools, but I can see it in the Xcode simulator.
